integration/01_circle_sha.go

```diff
@@ -0,0 +1,29 @@
+// +build k8srequired
+
+package integration
```

Comment:
Note the file name prefix. This is to guarantee initialization order of the compiler. I do not find that super nice but couldn't come up with something cool here. Ideas welcome.

Reply:
Why having separate files? I think a single init function in a single file would be more useful here. I also don't fully understand the need for all this code in the first place :) IMO environment variables are doing their job, no need for additional code to wrap them.

Reply:
I think having this sorted in code makes a lot of sense. There was some bash magic in the circle config to make up the cluster name and I think we can do better than this. Like, we have error handling in case something goes wrong which is explicit instead of some implicit failing which you have to hunt down by debugging more heavily in case shit hits the fan. I am fine with putting this into one go file though if this makes you happy.

Reply:
Not very much TBH :D it was just a suggestion for preventing the lexicographical naming restriction and the big warning comment.

Reply:
I need to know from you though which way you prefer so I can apply changes. I am not sure what to do now.

Reply:
Sorry, it's ok to continue with code if you feel that's better, if we can prevent having multiple files and the potential issues with initialization that's fine with me.

Reply:
Done.